<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FastFindClient</class>
 <widget class="QDialog" name="FastFindClient">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Быстрый поиск пациента</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="lblDescription">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Допустимые шаблоны поиска:&lt;/p&gt;&lt;p&gt;Ф* И* (Руд Его)&lt;/p&gt;&lt;p&gt;Ф* И* О* (Руд Его Евген)&lt;/p&gt;&lt;p&gt;Ф* И* О* ДДММГГ (Руд Его Евген 010295)&lt;/p&gt;&lt;p&gt;Ф* И* О* ДДММГГГГ (Руд Его Евген 01021995)&lt;/p&gt;&lt;p&gt;Ф* И* ДДММГГ (Руд Его 010295)&lt;/p&gt;&lt;p&gt;Ф* И* ДДММГГГГ (Руд Его 01021995)&lt;/p&gt;&lt;p&gt;Ф* ДДММГГ (Руд 010295)&lt;/p&gt;&lt;p&gt;Ф* ДДММГГГГ (Руд 01021995)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="edtTemplate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblTemplate">
     <property name="text">
      <string>&amp;Шаблон быстрого поиска</string>
     </property>
     <property name="buddy">
      <cstring>edtTemplate</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="btnSearch">
     <property name="text">
      <string>Искать</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="btnClose">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtTemplate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
